ui/components/SignInBackground.tsx

```diff
@@ -0,0 +1,23 @@
+// @ts-nocheck
```

Comment:
I moved the Lottie to its file so I can add the `ts-nocheck` without impacting too much code